src/core/experimental/memutils.d

```diff
@@ -0,0 +1,256 @@
+/**
+ * Pure D replacement of the C Standard Library basic memory building blocks of string.h
+ * Source: $(DRUNTIMESRC core/experimental/memutils.d)
+ */
+module core.experimental.memutils;
+
+/**
+ * If T is an array, set all `dst`'s bytes
+ * (whose count is the length of the array times
+ * the size of the array element) to `val`.
+ * Otherwise, set T.sizeof bytes to `val` starting from the address of `dst`.
+ * 
+ * Params
+ *  val = The byte with which we want to fill memory with.
+ *  dst = Memory Destination whose bytes are to be set to `val`.
+ *
+ * Returns:
+ *  Nothing.
```

Comment:
You can just leave the `Returns` section out if it's `void`. :wink:

Reply:
Haha, I didn't like it too. But I wanted a way to make a note that this `memset` is different from the libc in that it returns nothing. I'll add it as a `N.B.`.